apps/token-e2e/cypress.config.js

```diff
@@ -14,4 +14,20 @@ module.exports = defineConfig({
     screenshotsFolder: '../../dist/cypress/apps/explorer-e2e/screenshots',
     chromeWebSecurity: false,
   },
+  env: {
+    eth_provider_url: 'http://localhost:8545/',
+    eth_wallet_public_key: '0xEe7D375bcB50C26d52E1A4a472D8822A2A22d94F',
+    eth_staking_bridge_contract_address:
+      '0x9135f5afd6F055e731bca2348429482eE614CFfA',
+    vega_wallet_name: 'capsule_wallet',
+    vega_wallet_location: '~/.vegacapsule/testnet/wallet',
+    vega_wallet_passphrase: '123',
+    vega_wallet_mnemonic:
```

Comment:
Do we need define it in both env & fixtures? Also applies to passphrase, contract address & token address

Reply:
Good call, The wallet will not take a variable containing a passphrase or mnemonic.
It must exist as a file beforehand, so i created those files in the fixtures folder.
Elsewhere in other functions we need access to those values but i could change it so that we extract them from the fixtures files - and drop them from the env's, i'll action that point :)

The  contract address & token address feel more like env var's to me though?

Reply:
AFAIR as I remember discussion with @dexturr it's not likely to be changed in capsule, so we can keep it in fixtures (it's already there)